net.labymod.LabyMod.yml

```diff
@@ -0,0 +1,33 @@
+id: net.labymod.LabyMod
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: AppRun
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --env=ELECTRON_DISABLE_SANDBOX='1'
+  - --persist=.minecraft
+  #- --filesystem=home
+modules:
+  - name: LabyMod
+    buildsystem: simple
+    build-commands:
+       - unzip launcher.zip -d /app/bin/
+       - install -D --mode 0644 net.labymod.icon.png /app/share/icons/hicolor/scalable/apps/net.labymod.LabyMod.png
+       - install -D --mode 0644 net.labymod.LabyMod.desktop /app/share/applications/net.labymod.LabyMod.desktop
+       - install -D --mode 0644 net.labymod.LabyMod.metainfo.xml /app/share/metainfo/net.labymod.LabyMod.metainfo.xml
+    sources:
+      - type: file
```

Comment:
this should probably be `type: archive`

Reply:
I'll take a look as soon as the upload problem is solved.